pkg/decoder/decoder.go

```diff
@@ -196,7 +195,18 @@ func Parse(src []byte) (message.Message, error) {
 			Raw: make([]byte, len(src)),
 		}
 		copy(m.Raw, src)
-		//TODO
+
+		m.Relations = d.uint32()
```

Comment:
This field could be named relationsCount, but in fact we don't need it, as it's only useful to learn how many relationOids are there.